images/build/Dockerfiles/azureFunctions.JamStack.Dockerfile

```diff
@@ -11,32 +11,29 @@ ENV DEBIAN_FLAVOR=$DEBIAN_FLAVOR \
     PYTHONIOENCODING="UTF-8" \
     LANG="C.UTF-8" \
     LANGUAGE="C.UTF-8" \
-    LC_ALL="C.UTF-8"
-
-ARG IMAGES_DIR="/opt/tmp/images"
-RUN oryx prep --skip-detection --platforms-and-versions nodejs=14 --debug \
-    && echo "$DEBIAN_FLAVOR" \
-    && . /tmp/build/__goVersions.sh \
-    && downloadedFileName="go${GO_VERSION}.linux-amd64.tar.gz" \
-    && ${IMAGES_DIR}/retry.sh "curl -SLsO https://golang.org/dl/$downloadedFileName" \
-    && mkdir -p /usr/local \
-    && gzip -d $downloadedFileName \
-    && tar -xf "go${GO_VERSION}.linux-amd64.tar" -C /usr/local \
-    && rm -rf $downloadedFileName
-
+    LC_ALL="C.UTF-8" \
+    ORYX_PATHS="/opt/oryx:/opt/nodejs/lts/bin:/opt/python/latest/bin:/opt/yarn/stable/bin"
+    
 RUN set -ex \
     # Install Python SDKs
     # Upgrade system python
     && PYTHONIOENCODING="UTF-8" \
+    # It's not clear whether these are needed at runtime...
```

Comment:
Could you please add more info about this? From what I've seen in the past I needed to update apt-get before using it for it to be able to detect and pull the latest packages. However it also came with a significant cost, as it seems that updating it pulls A LOT of data. If you have any more info, it could be valuable so that we update that code later or something.

Reply:
Seems like this [this](https://github.com/microsoft/Oryx/blob/180b2a86eaa248433d7b7f3defbdf112026356d3/images/build/Dockerfiles/azureFunctions.JamStack.Dockerfile#L44) comment has been around for a while. It is not clear why it was added from the PR.

Reply:
OK thanks. One of the things to investigate separately, but I agree it doesn't need to be done as part of this PR.